foundations/likvid-prod/platforms/azure/meshplatform-au/terragrunt.hcl

```diff
@@ -0,0 +1,67 @@
+include "common" {
+  path = find_in_parent_folders("common.hcl")
+}
+
+include "platform" {
+  path   = find_in_parent_folders("platform.hcl")
+  expose = true
+}
+
+dependency "bootstrap" {
+  config_path = "../bootstrap"
+}
+
+dependency "au-cloud-native" {
+  config_path = "../landingzones/au-cloud-native"
+}
+
+terraform {
+  source = "${get_repo_root()}//kit/azure/meshplatform-au"
+}
+
+generate "provider" {
+  path      = "provider.tf"
+  if_exists = "overwrite"
+  contents  = <<EOF
+provider "azurerm" {
+  features {}
+  resource_provider_registrations = "none"
+  tenant_id       = "${include.platform.locals.platform.azure.aadTenantId}"
+  subscription_id = "${include.platform.locals.platform.azure.subscriptionId}"
+}
+
+provider "azapi" {
+  skip_provider_registration = false
+  tenant_id       = "${include.platform.locals.platform.azure.aadTenantId}"
+  subscription_id = "${include.platform.locals.platform.azure.subscriptionId}"
+}
```

Comment:
Iam not 100% sure but i guess we dont use the azapi provider here. I added it before but iirc we dont use it :/